src/_posts/compliance/2000-01-01-hds.md

```diff
@@ -70,3 +76,9 @@ La certification HDS de Scalingo couvre les périmètres suivants :
 Veuillez toujours vous référer à la page actuelle pour obtenir les dernières informations sur notre conformité HDS
 
 [Télécharger le certificat HDS 1.1 en vigueur](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
+
+## 🇫🇷 Qu'est-ce qu'un "Point de contact professionnel de santé" ? {#fr-hds-point-of-contact-health-professional}
+
+La certification HDS impose à Scalingo de maintenir une liste de *points de contact professionnel de santé* pour chacun de ses clients.
+Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer des relations avec le patient, etc.)
+En outre, Scalingo doit être en capacité de transmettre sans délai cette liste à l’autorité compétente sur demande, notamment en cas de suspension ou de retrait de la certification HDS.
```

Comment:
Source ?

Reply:
Page 17 of this pdf file, which seems legit: https://esante.gouv.fr/sites/default/files/media_entity/documents/hds_referentiel_de_certification_v1.1f_mai2018.pdf

Please tell me if I should use another source :)

Reply:
You couldn't find better source in fact :)
I suggest you mention it in the french and english versions (there is an english translation of the referential too)

Reference it like this

> En outre, Scalingo doit être en capacité de transmettre sans délai cette liste à l’autorité compétente sur demande, notamment en cas de suspension ou de retrait de la certification HDS (Référentiel HDS 1.1.1f, exigence 4.5.4).

Reply:
Should I add a link the referential? (if so, could you please provide them?)

Reply:
@Frzk IMHO linking the referential is not necessary. But you should mention the referential article number like Yannick did ie "Référentiel HDS 1.1.1f, exigence 4.5.4"

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/d5252491897da1d156b7e0e5ea7e81b8e7027781